Tests/scripts/shutdown_instance.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-#in case circle didn't terminate instance for some reason
-shutdown +150
```

Comment:
why are we removing this @Bars92 ?

Reply:
This is embedded in the https://github.com/demisto/content-test-conf/blob/master/instance.json file under user data instead of a script